Cdn_BunnyCdn_Widget_View_Unauthorized.php

```diff
@@ -13,7 +14,105 @@
 ?>
 
 <div id="bunnycdn-widget" class="w3tc_bunnycdn_signup">
-	<?php if ( ! $c->get_boolean( 'cdn.enabled' ) ) : ?>
+	<?php
+	$cdn_engine = $c->get_string( 'cdn.engine' );
+	$cdn_name   = '';
```

Comment:
Get the CDN name from https://github.com/BoldGrid/w3-total-cache/blob/2.8.2/Cache.php#L103 and not duplicate code using a new switch/case statement.

Reply:
Changed to use Cache::engine_name and updated logic to check if cdn_engine is not empty so that the notice doesn't display as "detected none"